Infrastructure/gpg-metrics/gpg-prometheus/manifest.yml

```diff
@@ -1,9 +1,12 @@
 ---
 applications:
   - name: gpg-prometheus
-    memory: 512M
+    memory: 3G

```

Comment:
I checked the app details before making any changes and the memory size was 3G.

However, during the initial deployment it has been set to 512M; the app (memory) might have been updated manually without doing a deployment.

Reply:
Is there any cost increase, since we've increased the available memory?

Reply:
ah I didn't increase the memory. This is what happened:
1. App deployed using this manifest file - memory = 512M
2. App memory increased to 3G manually (by running a command, without having another deployment) => manifest file not updated

I've just updated the `manifest.yml` to reflect the actual values

Reply:
Oh, I see, that makes sense 😅. It all looks good to me then!